examples/pipeline_wav2letter/transforms.py

```diff
@@ -0,0 +1,11 @@
+import torch
+
+
+class Normalize(torch.nn.Module):
+    def forward(self, tensor):
+        return (tensor - tensor.mean(-1, keepdim=True)) / tensor.std(-1, keepdim=True)
+
+
+class UnsqueezeFirst(torch.nn.Module):
+    def forward(self, tensor):
+        return tensor.unsqueeze(0)
```

Comment:
This is, in my opinion, the sort of issue that makes me dislike using nn.Sequential. You end up wrapping simple, small commands into modules.